contrib/opencollective_hookshot.js

```diff
@@ -0,0 +1,75 @@
+// Input is data object which is hook body
+// Output is result variable of form
+// {
+//     "version": "v2" // The version of the schema being returned from the function. This is always "v2".
+//     "empty": true|false, // Should the webhook be ignored and no output returned. The default is false (plain must be provided).
+//     "plain": "Some text", // The plaintext value to be used for the Matrix message.
+//     "html": "<b>Some</b> text", // The HTML value to be used for the Matrix message. If not provided, plain will be interpreted as markdown.
+//     "msgtype": "some.type", // The message type, such as m.notice or m.text, to be used for the Matrix message. If not provided, m.notice will be used.
+// }
+
+// Configure this for the collective you are receiving webhooks from.
+// Expense payloads (unlike transactions) don't have this in the data.
+const defaultCollective = {
+    slug: "sunpy",
+    name: "SunPy"
+};
+
+const expenseNameBy = function(expense, user) {
+    return `${expense.description} for ${expense.formattedAmount} by ${user.name}`;
+};
+
+const expenseNameByHtml = function(expense, user) {
+    return `<a href="https://opencollective.com/${defaultCollective.slug}/expenses/${expense.id}">${expense.description}</a> for <b>${expense.formattedAmount}</b> by <a href="https://opencollective.com/${user.slug}">${user.name}</a>`;
+};
+
+const collectiveExpenseAction = function(expense, user, verb) {
+    return {
+        plain: `Expense ${verb}: ${expenseNameBy(expense, user)}`,
+        html: `<b>Expense ${verb}</b>: ${expenseNameByHtml(expense, user)}`
+    };
+};
+
+const expenseVerbs = {
+    "collective.expense.created": "Created",
+    "collective.expense.approved": "Approved",
+    "collective.expense.paid": "Paid",
+    // This is a guess I haven't rejected an expense yet.
+    "collective.expense.rejected": "Rejected"
+};
+
+const capitalise = function(string) {
+    string = string.toLowerCase();
+    return string[0].toUpperCase() + string.substring(1);
```

Comment:
Current code throws for an empty string.

```suggestion
    return string.slice(0,1).toUpperCase() + string.slice(1).toLowerCase();
```